lib/xclarity_client/services/update_repo_management.rb

```diff
@@ -2,12 +2,12 @@ module XClarityClient
   class UpdateRepoManagement < Services::XClarityService
     manages_endpoint UpdateRepo
 
-    def population(opts = {})
+    def fetch_all(opts = {})
       raise "Option key must be provided for update_repo resource" if opts.empty?
       raise "Option key must be provided for update_repo resource" if not (opts.has_key?(:key) || opts.has_key?("key"))
       repoKey = opts[:key] || opts["key"]
       if repoKey == "supportedMts" || repoKey == "size" || repoKey == "lastRefreshed" || repoKey == "importDir" || repoKey == "publicKeys" || repoKey == "updates" || repoKey == "updatesByMt" || repoKey == "updatesByMtByComp"
```

Comment:
Just an idea to improve this would be to use %w(supportedMts size ...).include?(repoKey)

Reply:
Good point. But I believe that is better to do this refactor in another PR, once it will runaway of the main purpose of this one.